Jenkinsfile

```
@@ -3,6 +3,7 @@ pipeline {
 
     options {
         timeout(time: 60, unit: 'MINUTES')
+        disableConcurrentBuilds()
```

Do we want to abort previous runs instead? Similar to GH actions?

```suggestion
        disableConcurrentBuilds(abortPrevious: true)
```

https://www.jenkins.io/doc/book/pipeline/syntax/#options

---

good catch